README.md

```diff
@@ -0,0 +1,22 @@
+# WiseTech Global Analyzers
+
+[![Build](https://img.shields.io/appveyor/ci/WiseTechGlobal/wtg-analyzers/master.svg)](https://ci.appveyor.com/project/WiseTechGlobal/wtg-analyzersd)
+[![Tests](https://img.shields.io/appveyor/tests/WiseTechGlobal/wtg-analyzers/master.svg)](https://ci.appveyor.com/project/WiseTechGlobal/wtg-analyzersd)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/WiseTechGlobal/WTG.Analyzers/blob/master/LICENSE.md)
+[![NuGet](https://img.shields.io/nuget/v/WTG.Analyzers.svg)](http://nuget.org/packages/WTG.Analyzers)
+
+Roslyn Analyzers from [WiseTech Global](https://www.wisetechglobal.com/) to enforce our styles, behaviours, and prevent common mistakes.
+
+**Note for WTG staff:** This repository is only for rules that are generic enough to apply to any .NET project. For rules that are specific to internal projects and frameworks, create an internal Analyzers project.
```

Comment:
I'm not sure if it's worth directing comments towards staff, internal staff will probably send an email rather than visit this site :)

Reply:
Lead With Content 😉